getting-started/configure-authentication.md

```diff
@@ -21,7 +21,7 @@ After registration, you should have a `client id` and `client secret` value for
 
     **NOTE:** If you are not using Helm to manage secrets, you will need to configure the OpenID Connect secret manually as described in the [secrets document](secrets.md).
 
-3. \[optional\] In _systemlink-values.yaml_, configure the `webserver.oidc.scope` value to select the OpenID Connect scopes that SystemLink Enterprise will request. By default, SystemLink Enterprise requests the "openid", "email", and "profile" scopes. The "openid" scope is required. The "profile" and "email" scopes are used to populate user details in the UI and other scopes may be useful when assigning user roles in the application. Consult the documentation for your authentication provider to see what scopes are available.
+3. \[optional\] In _systemlink-values.yaml_, configure the `webserver.oidc.scope` value to select the OpenID Connect scopes that SystemLink Enterprise will request. By default, SystemLink Enterprise requests the "openid", "email", and "profile" scopes. The "openid" scope is required. The "profile" and "email" scopes are used to populate user details in the UI and other scopes may be useful when assigning user roles in the application. The "offline_access" scope grants access to the users' information at `/oidc/userinfo`. Consult the documentation for your authentication provider to see what scopes are available.
```

Comment:
Is this worth moving to its own section, similar to https://operations.systemlink.io/openid-connect/openid-connect/#viewing-claims-returned-by-a-provider

Reply:
@TrevorKarjanis Can we resolve this PR prior to this weeks release?